src/output.rs

```diff
@@ -26,17 +27,15 @@ pub fn print_page(
     if enable_markdown {
         // Print the raw markdown of the file.
         for line in reader.lines() {
-            writeln!(handle, "{}", line.unwrap())
-                .map_err(|_| "Could not write to stdout".to_string())?;
+            writeln!(handle, "{}", line.unwrap()).context("Could not write to stdout")?;
```

Comment:
What do you mean with handling? In #247 we're simply throwing an error to avoid a panic, but there's no handling. Here we throw an error as well.

In any case, a failure to write to stdout is _highly_ improbable. This is actually a line that could even be handled with a panic (but I guess the error is OK as well).

Reply:
No, I mean the `line.unwrap` which we could now also make an error instead of a panic. Like I said, I am not rooting for either side here though

Reply:
Ah, I see. Well, it certainly doesn't hurt 🙂